.github/workflows/build_binaries_on_new_releases.yml

```diff
@@ -29,15 +29,14 @@ jobs:
         - name: windows
           os: windows-2022
           target: x86_64-pc-windows-msvc
-
+    permissions:
+      contents: write
```

Comment:
I understand that we need to grant permissions to release binary. Do you mind if you add comment for explanation why we define permissions just in case?
https://github.com/softprops/action-gh-release#permissions

Reply:
It makes sense. I added the comment for the permission.